### PR TITLE
Bazel to CMake: Add further mappings

### PR DESCRIPTION
**Pull request by marbre**

### CODE REVIEW

File: build_tools/scripts/bazel_to_cmake_targets.py [hunk 1]
```diff
@@ -123,7 +134,12 @@ def convert_external_target(target):
   if target.startswith("@org_tensorflow//tensorflow/lite/experimental/ruy"):
     # All Bazel targets map to a single CMake target.
     return "ruy"
+  if target.startswith("@sdl2//:SDL2"):
+    return "SDL2-static"
   if target.startswith("@vulkan_headers"):
     return VULKAN_HEADERS_MAPPING[target]
+  if target.startswith("@vulkan_sdk//:sdk"):
+    # The Bazel target maps to the IMPORTED target defined by FundVulkan().
```
Comment (ScottTodd) [inline build_tools/scripts/bazel_to_cmake_targets.py]: typo: FindVulkan

Comment (marbre) [reply build_tools/scripts/bazel_to_cmake_targets.py]: Fixed.

File: build_tools/scripts/bazel_to_cmake_targets.py [hunk 2]
```diff
@@ -36,9 +36,14 @@ def _convert_absl_target(target):
     target_name = target.rsplit("/")[-1]
   return "absl::" + target_name
 
+DEAR_IMGUI_EXPLICIT_TARGET_MAPPING = {
+  "@dear_imgui": "dear_imgui::dear_imgui",
+  "@dear_imgui//:imgui_sdl_vulkan": "dear_imgui::impl_sdl\n    dear_imgui::impl_vulkan",
```
Comment (GMNGeoffrey) [inline build_tools/scripts/bazel_to_cmake_targets.py]: Can we instead change these mappings to be one to many so we don't have to do weird string manipulation here? Map targets would return a list which the caller would be responsible for formatting properly. Could be a follow-up

File: build_tools/scripts/bazel_to_cmake_targets.py [hunk 3]
```diff
@@ -123,7 +134,12 @@ def convert_external_target(target):
   if target.startswith("@org_tensorflow//tensorflow/lite/experimental/ruy"):
     # All Bazel targets map to a single CMake target.
     return "ruy"
+  if target.startswith("@sdl2//:SDL2"):
```
Comment (GMNGeoffrey) [inline build_tools/scripts/bazel_to_cmake_targets.py]: This should be an equals, I think.

File: build_tools/scripts/bazel_to_cmake_targets.py [hunk 4]
```diff
@@ -123,7 +134,12 @@ def convert_external_target(target):
...
+    return "SDL2-static"
   if target.startswith("@vulkan_headers"):
     return VULKAN_HEADERS_MAPPING[target]
+  if target.startswith("@vulkan_sdk//:sdk"):
```
Comment (GMNGeoffrey) [inline build_tools/scripts/bazel_to_cmake_targets.py]: As should this

File: build_tools/scripts/bazel_to_cmake_targets.py (hunk 3 above)
Comment (marbre) [reply build_tools/scripts/bazel_to_cmake_targets.py]: Correct. Fixed.

File: build_tools/scripts/bazel_to_cmake_targets.py (hunk 4 above)
Comment (marbre) [reply build_tools/scripts/bazel_to_cmake_targets.py]: Fixed as well.

File: build_tools/scripts/bazel_to_cmake_targets.py (hunk 2 above)
Comment (marbre) [reply build_tools/scripts/bazel_to_cmake_targets.py]: I like the idea, but I wasn't successful with a one liner that transforms only sublists. So would prefer a follow-up.

Comment (GMNGeoffrey) [reply build_tools/scripts/bazel_to_cmake_targets.py]: Sounds good. I think I would just make all the convert_target methods return lists

**Pull request closed without merging**